docs/DevGuide/Travis.md

```diff
@@ -74,6 +74,26 @@ performed in the CHECK_FILES build.
 using a Linux OS, and the `AppleClang` compiler for `OS X`.
 * The `gcc Debug` build will fail if there are `doxygen` warnings.
 
+## How to perform the checks locally
+
+Before pushing to GitHub and waiting for Travis to perform the checks it is
+useful to perform at least the following tests locally:
+- **Unit tests:** Perform a `make test-executables` and then `ctest -L unit` to
+  run all unit tests. This can take a significant amount of time, so to run only
+  a subset of the tests you can use one of the other keywords that the tests are
+  labeled with, such as `ctest -L datastructures`. To run only a particular test
+  you can also execute `BUILD_DIR/bin/RunTests TEST_NAME` instead, where
```

Comment:
ctest -R TEST_NAME to run it as a test (i.e. ERROR_TESTS that successfully trigger an error result in the test passing).   You might also want to note that the output of running ctest is in BUILD_DIR/Testing/Temporary/LastTest.log